Added information about UTC time format (#9746)
MicrosoftDocs/office-docs-powershell: exchange/exchange-ps/exchange/Get-MessageTrace.md

By default, this cmdlet returns a maximum of 1000 results, and will timeout on very large queries. If your query returns too many results, consider splitting it up using smaller StartDate and EndDate intervals.

> The time stamps on the output are in UTC time format. That might be different time format than you were using for the -StartDate and the -EndDate parameters.

Reviewer: ```suggestion
The time stamps on the output are in UTC time format. That might be different from the time format that you used for the -StartDate and the -EndDate parameters.
```
